tutorials/wheel_slip_systems.md

```diff
@@ -0,0 +1,138 @@
+\page wheel_slip_systems Wheel Slip systems
+
+# Overview
+
+This tutorial shows how to configure slip parameters for wheeled vehicles using
+two different wheel slip plugins.
+
+Wheel slip configuration is useful for simulating scenarios where a vehicle
+drives over different types of surfaces. For example, a vehicle climbing up
+steep gravel road may start slipping and the Gazebo allows users to configure
+the slip compliance and friction parameters of each individual wheel to
+approximate this behavior instead of simulating each individual rock on the
+road and its contact with the wheel.
+
+# Wheel Slip systems
+
+There are two systems in Gazebo related to wheel slip configuration:
+
+1. Wheel slip system
+1. Lookup wheel slip system
+
+Both of the these systems are model plugins, meaning they should be added to a
+model in the world. The wheel slip system allows users to configure slip
+parameters for each wheel in the model. The lookup wheel slip system on the
+other hand has to be used together with the wheel slip system. It provides extra
+functionality that enables the slip parameters to be dynamically updated as the
+vehicle moves around in the environment.
+
+
+## Wheel Slip system
+
+The wheel slip system updates wheel slip parameters based on linear wheel spin
+velocity. Here are the parameters that can be configured per wheel link:
+
+* `<slip_compliance_lateral>`: Unitless wheel slip compliance in lateral
+  direction. The parameter should be non-negative, with a value of zero allowing
+  no slip and larger values allowing increasing slip.
+* `<slip_compliance_longitudinal>`: Unitless wheel slip compliance in longitudinal
+  direction. The parameter should also be non-negative and larger values mean
+  increasing slip.
+* `<wheel_normal_force>`: Wheel normal force estimate used to compute slip
+  compliance, which takes units of 1/N.
+* `<wheel_radius>`: Radius of the wheel in meters.
+
+You can find more detailed information on the wheel slip system and its
+parameters in the
+[WheelSlip.hh](https://github.com/gazebosim/gz-sim/blob/gz-sim10/src/systems/wheel_slip/WheelSlip.hh)
+header file.
+
+Gazebo has an example world that demonstrates the use of the wheel slip system,
+namely [trisphere_cycle_wheel_slip.sdf](https://github.com/gazebosim/gz-sim/blob/gz-sim10/examples/worlds/trisphere_cycle_wheel_slip.sdf).
```

Comment:
we should update this example world to include joint controllers

for example, the [plowing_effect_tricycle_demo.world](https://github.com/gazebosim/gazebo-classic/blob/b22c6e15e52299865b31093b8feebc9ca19e26e8/test/worlds/plowing_effect_tricycle_demo.world#L59-L77) has position controllers on the steering joint and velocity controllers on the rear wheel joints. This will cause the robots to drive forward and you can see the effect of wheel slip